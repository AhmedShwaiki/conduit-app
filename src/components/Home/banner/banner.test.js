import toJson from "enzyme-to-json";
import { shallow } from "enzyme";
import { React } from "react";
import Banner from "./banner";

it("renders correctly", () => {
  const wrapper = shallow(<Banner />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
