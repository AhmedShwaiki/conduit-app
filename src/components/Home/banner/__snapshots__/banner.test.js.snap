// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<styled.div>
  <styled.div>
    conduit
  </styled.div>
  <styled.p>
    A place to share your knowledge
  </styled.p>
</styled.div>
`;
